File: java/src/main/java/com/experian/dvs/client/address/format/enrichment/GbrBusiness.java
package com.experian.dvs.client.address.format.enrichment;

import com.experian.dvs.client.server.address.format.RestApiEnrichmentGbAdditional;

import java.util.Objects;

public class GbrBusiness {

    private final String urn;
    private final CommercialMosaicElements commercialMosaic;
    private final String registration;
    private final String nonLimitedCompanyKey;
    private final String phone;
    private final String numberOfEmployees;
    private final StandardIndustryClassificationElements standardIndustryClassification;
    private final LocationElements location;

    // Constructor
    public GbrBusiness(RestApiEnrichmentGbAdditional api) {
        this.urn = Objects.toString(api.getUrn(), "");
        this.commercialMosaic = api.getCommercialMosaic() != null ? new CommercialMosaicElements(api.getCommercialMosaic()) : null;
        this.registration = Objects.toString(api.getRegistration(), "");
        this.nonLimitedCompanyKey = Objects.toString(api.getNonLimitedCompanyKey(), "");
        this.phone = Objects.toString(api.getPhone(), "");
        this.numberOfEmployees = Objects.toString(api.getNumberOfEmployees(), "");
        this.standardIndustryClassification = api.getStandardIndustryClassification() != null ? new StandardIndustryClassificationElements(api.getStandardIndustryClassification()) : null;
        this.location = api.getLocation() != null ? new LocationElements(api.getLocation()) : null;
    }

    // Getters
    public String getUrn() {
        return urn;
    }

    public CommercialMosaicElements getCommercialMosaic() {
        return commercialMosaic;
    }

    public String getRegistration() {
        return registration;
    }

    public String getNonLimitedCompanyKey() {
        return nonLimitedCompanyKey;
    }

    public String getPhone() {
        return phone;
    }

    public String getNumberOfEmployees() {
        return numberOfEmployees;
    }

    public StandardIndustryClassificationElements getStandardIndustryClassification() {
        return standardIndustryClassification;
    }

    public LocationElements getLocation() {
        return location;
    }
}

File: csharp/DVSClient/DVSClient/address/format/enrichment/GbrBusiness.cs
using DVSClient.Server.Address.Format;

namespace DVSClient.Address.Format.Enrichment
{
    public class GbrBusiness
    {
        public string? Urn { get; }
        public CommercialMosaicElements? CommercialMosaic { get; }
        public string? Registration { get; }
        public string? NonLimitedCompanyKey { get; }
        public string? Phone { get; }
        public string? NumberOfEmployees { get; }
        public StandardIndustryClassificationElements? StandardIndustryClassification { get; }
        public LocationElements? Location { get; }

        public GbrBusiness(RestApiEnrichmentGbAdditional api)
        {
            Urn = api.Urn ?? null;
            CommercialMosaic = api.CommercialMosaic != null ? new CommercialMosaicElements(api.CommercialMosaic) : null;
            Registration = api.Registration ?? null;
            NonLimitedCompanyKey = api.NonLimitedCompanyKey ?? null;
            Phone = api.Phone ?? null;
            NumberOfEmployees = api.NumberOfEmployees ?? null;
            StandardIndustryClassification = api.StandardIndustryClassification != null ? new StandardIndustryClassificationElements(api.StandardIndustryClassification) : null;
            Location = api.Location != null ? new LocationElements(api.Location) : null;
        }
    }
}